stablehlo/transforms/Passes.td

```diff
@@ -79,6 +79,25 @@ def StablehloRefineShapesPass : Pass<"stablehlo-refine-shapes", "ModuleOp"> {
   }];
 }
 
+def StablehloRefineParametersPass : Pass<"stablehlo-refine-parameters", "ModuleOp"> {
```

Comment:
To my understanding running this pass on its own doesn't make a whole lot of sense, as it will introduce calls to a function that doesn't exist, and these should be cleaned up. So if you're running this pass, shape refinement should also run, probably right after this pass. I wonder if there's a way to enforce that? Or maybe we should make that explicit in the summary and description.

Reply:
We can provide a pass pipeline, but I think keeping the responsibilities separates is best for testability.

Reply:
Calling that a followup, I think once we get to integrating this pass we can get a better idea on that.